### ISSUE TITLE
Add LOD and LOQ to Statistical Values

### ISSUE THREAD

**Issue by Sakshi-Handa** [HumanExposure/factotum#2141]
As a product owner 
I would like to move the LOD and LOQ fields to the Statistical Value list
So that I can store additional data, including units and reported/calculated fields for the LOD/LOQ

When reporting the LOD and LOQ, we would like to be able to store additional data related to the units and whether the data was reported or calculated. Currently this data is stored as a numeric field, but we would like to add it to the 'statistical values' to be able to capture more detail.
Ex. LM doc https://ccte-factotum-stg.epa.gov/datadocument/1555997/
Ex. HHE doc https://ccte-factotum.epa.gov/datadocument/1367747/

Definition of Done
	• Applies to the LM and HHE data models
	• Remove the numeric fields for LOD (limit of detection) and LOQ (limit of quantification)
Add LOD and LOQ as options for 'measurement name' under "Statistical Values"

**Comment by mwfrost** [comment]
I am hoping to fix #2145 on this branch, since the upload functionality is related. 

When users upload cleaned data with a `statistical_value` column, the data in that column looks like this (example borrowed from the 3M test file in that ticket):
```
{'name': 'max', 'value': 19, 'value_type': 'R', 'stat_unit': 'mg/L'}; {'name': '75th Percentile', 'value': 10, 'value_type': 'C', 'stat_unit': 'mg/L'}; 
```
The notation used for the "name" value is too picky to allow both "max" (uncapitalized) and "75th Percentile". Currently all the "name" strings must be in the capitalized keys below:
```
    ("MEAN", "Mean"),
    ("MEDIAN", "Median"),
    ("MAX", "Max"),
    ("P75", "75th Percentile"),
    ("P95", "95th Percentile"),
    ("P99", "99th Percentile"),
    ("SE", "Standard Error"),
    ("SD", "Standard Deviation"),
    ("LOD", "Limit of Detection"),
    ("LOQ", "Limit of Quantitation"),
    ```
The system is forgiving enough to convert "mean" to "MEAN" if the user submits something uncapitalized, but mixing "75th Percentile" and "LOD" does not work. The csv file has to either use "P75" and "LOD" or we can change it to use the full human-friendly strings like "99th Percentile" and "Limit of Quantitation". Whatever we settle on, I can add a note to the upload form.

**Comment by Sakshi-Handa** [comment]
I would say we can require use of the shorter 'codes' for upload, and require all caps. It would easier than typing out the full name for a lot of these. As long as we know the rules for formatting, we can adapt our scripts to fit.
